### ISSUE TITLE
error: use of undeclared identifiers (`half2half2` and `h2div`)

### ISSUE THREAD

**Issue by acxz** [rocm-arch/python-pytorch-rocm#34]
`half2half2`
```
/home/acxz/vcs/git/github/rocm-arch/python-pytorch-rocm/src/pytorch-1.11.0rc2-rocm/cmake/../third_party/eigen/Eigen/src/Core/arch/GPU/PacketMathHalf.h:51:10: error: use of undeclared identifier 'half2half2'; did you mean '__half2half2'?
  return half2half2(from);
         ^~~~~~~~~~
         __half2half2
/opt/rocm/hip/include/hip/amd_detail/amd_hip_fp16.h:550:21: note: '__half2half2' declared here
            __half2 __half2half2(__half x)
                    ^
```

`h2div`:
```
/home/acxz/vcs/git/github/rocm-arch/python-pytorch-rocm/src/pytorch-1.11.0rc2-rocm/cmake/../third_party/eigen/Eigen/src/Core/arch/GPU/PacketMathHalf.h:292:10: error: use of undeclared identifier 'h2div'
  return h2div(a, b);
         ^
2 errors generated when compiling for gfx803.
CMake Error at torch_hip_generated_gelu_op.hip.o.cmake:200 (message):
  Error generating file
  /home/acxz/vcs/git/github/rocm-arch/python-pytorch-rocm/src/pytorch-1.11.0rc2-rocm/build/caffe2/CMakeFiles/torch_hip.dir/operators/hip/./torch_hip_generated_gelu_op.hip.o
```

**Comment by acxz** [comment]
Upstream issue: https://github.com/pytorch/pytorch/issues/73177

**Comment by acxz** [comment]
patched with: [aa3e962](https://github.com/rocm-arch/python-pytorch-rocm/commit/aa3e96229c10a903caa039dfea0b1b1b1a8cf6f8)
